_generator/sorting/operationSort.js

```diff
@@ -43,5 +44,8 @@ function getOperationPriority(operation) {
   if (!priority && sortKey.startsWith('get')) {
     priority = sortPriority.get;
   }
+  if (!priority && sortKey.startsWith('add')) {
+    priority = sortPriority.add + 0.5;
```

Comment:
note: While it looks weird, this way we should make sure that `add` is sorted before `addSomething`.